docs/proposal/1-authentication-support/README.md

```diff
@@ -0,0 +1,175 @@
+# kstone-api Supports Authentication
+
+## Motivation
+
+Kstone has become a popular solution for etcd management. Due to lack of authentication ability, it has safety vulnerability and is not recommended using in production environment. To improve safety features of kstone, we'd like to support authentication in kstone.
+
+## Goals
+
+1. Use username/password authentication to implement login feature(create an admin user by default). 
+2. Support extendable authentication protocol (e.g. ldap, oidc).
+ 
+## Non-Goals
+
+Implement a complex RBAC authentication system.
+
+## Use Cases
+
+Visiting [kstone-dashboard](https://github.com/tkestack/kstone-dashboard) home page, it redirects to the login page.
+
+### Basic Auth
```

Comment:
https://en.wikipedia.org/wiki/Basic_access_authentication
https://stackoverflow.com/questions/34013299/web-api-authentication-basic-vs-bearer
It is easy to get confused here, we should use bearer token authentication.